app/cmd/config_generate.go

```diff
@@ -84,7 +84,8 @@ func printCfg(data []byte) {
 
 func generateSampleConfig() ([]byte, error) {
 	sampleConfig := Config{
-		Current: "yourServer",
+		Current:  "yourServer",
+		Language: "en",
```

Comment:
I prefer no default value here. It's best to get the default value from os platform.

Reply:
I can't use GoLang to get the system language information directly, I need to intercept it from the system environment variables. And the intercepted value can not be used directly, which is more cumbersome for the program, I feel not a good thing

Reply:
It's ok that no default language getting from the system. But we don't need to give it default as English.

Reply:
Yeah. I will remove the default `en` ready